connect-and-run.md

```diff
@@ -128,36 +146,41 @@ Note: the default path to the SAS executable (saspath) is /opt/sasinside/SASHome
 
-### Profile Anatomy (SAS 9.4 Remote)
-
-The parameters listed below make up the profile settings for configuring a connection to a remote SAS 9.4 instance.
-
-| Name         | Description                          | Additional Notes                                                     |
-| ------------ | ------------------------------------ | -------------------------------------------------------------------- |
-| **Name**     | Name of the profile                  | This will display on the status bar                                  |
-| **Host**     | SSH Server Host                      | This will appear when hovering over the status bar                   |
-| **Username** | SSH Server Username                  | A username to use when establishing the SSH connection to the server |
-| **Port**     | SSH Server Port                      | The ssh port of the SSH server. Default value is 22                  |
-| **SAS Path** | Path to SAS Executable on the server | Must be a fully qualified path on the SSH server to a SAS executable |
-
-## Add New SAS 9.4 Remote Profile
-
-Open the command palette (`F1`, or `Ctrl+Shift+P` on Windows or Linux, or `Shift+CMD+P` on OSX). After executing the `SAS.addProfile` command, select the SAS 9.4 (remote) connection type and complete the prompts (using values from the preceeding table) to create a new profile.
-
-## Profile Anatomy (SAS 9.4 local)
+## Profile: SAS 9.4 (local)
 
 On Windows, during the install of SAS 9.4, make sure that "Integration Technologies Client" checkbox is checked. If using an order that doesn't provide this option, make sure the ITC is installed by visiting the following [link](https://support.sas.com/downloads/browse.htm?fil=&cat=56). Make sure to download the 9.4m8 option.
 
+### Profile Anatomy
+
 The parameters listed below make up the profile settings for configuring a connection to a local SAS 9.4 instance.
 
 | Name     | Description                      | Additional Notes                    |
 | -------- | -------------------------------- | ----------------------------------- |
 | **Name** | Name of the profile              | This will display on the status bar |
 | **Host** | Indicates SAS 9.4 local instance | Defaults to localhost for com       |
 
-## Add New SAS 9.4 Local Profile
+### Add New SAS 9.4 (local) Profile
+
+Open the command palette (`F1`, or `Ctrl+Shift+P` on Windows). After executing the `SAS.addProfile` command, select the SAS 9.4 (local) connection type to create a new profile.
+
+## Profile: SAS 9.4 (remote - IOM)
+
+In order to use this option, you'll need to have "Integration Technologies Client" (ITC) installed. You can do this by making sure the "Integration Technologies Client" checkbox is checked when installing SAS 9.4, or by visiting the following [link](https://support.sas.com/downloads/browse.htm?fil=&cat=56) (Make sure to download the 9.4m8 option).
```

Comment:
nit: as my comment for "Windows only", it would be good to make it clear that the ITC has to be installed on the machine that our extension runs on. Not on the SAS 9.4 server, nor the VS Code UI if the user is tunneling.